src/app/page.tsx

```diff
@@ -3,9 +3,16 @@ import styles from './page.module.css';
 
 export default function Home() {
   return (
-    <main className={styles.main}>
-      <h1>Welcome to Deer Tracker!</h1>
-      <h3>Powered by AI & Machine Learning</h3>
+    <main className={styles.mainpageContainer}>
+      <div className={styles.headerconatiner}>
+        <header className={styles.h1}>Welcome to Deer Tracker!</header>
+      </div>
+      <div className={styles.lgnbttncontainer}>
+        <button className={styles.loginbtton}>login</button>
+      </div>
```

Comment:
Move the login button into a seperate component file called ActiveButton so you can reuse this button across multiple pages easily. be sure to follow DRY (Do Not Repeat Yourself) while coding as much as you can

Also change text from "login" to "Log in"